src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/MergeSigningInfo.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Tests
+{
+    [TestFixture]
+    public class MergeSigningInfo
+    {
+        private PushMetadataToBuildAssetRegistry pushMetadata;
+
+        public MergeSigningInfo()
```

Comment:
Instead of a constructor, consider using the [Setup] attribute on a method here. It will run per each test.

Reply:
Theoretically it shouldn't make a difference, since there's no state change, so I was trying to cut down on per-test setup, but there didn't end up being much logic there anyway so I can do this.

Reply:
Done